canonical-web-and-design/multipass.run
# Add Cheat Sheet modal to multipass homepage

## Done

- Update node-sass to sass
- Add cheat sheet section to landing page
- Add pop up modal to collect user info when downloading cheat sheet 
- **Blocked**: After entering details no download doesn't start

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8026/
- Check the cheat sheet downloads when the button is clicked
- [Copy doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#)


## Issue / Card

Fixes #139
